qiita_pet/uimodules/raw_data_tab.py

```diff
@@ -212,14 +174,22 @@ def render(self, prep, study_id, is_editable, ena_terms,
         preprocessing_status = prep.preprocessing_status
 
         if raw_data.filetype in ('SFF', 'FASTA'):
-            preprocess_form = Preprocess454ParametersForm()
+            param_iter = Preprocessed454Params.iter()
         elif raw_data.filetype == 'FASTQ':
-            preprocess_form = PreprocessIlluminaParametersForm()
+            param_iter = PreprocessedIlluminaParams.iter()
         else:
             raise ValueError("Don't know what to do but this exception will "
                              "never actually get shown anywhere because why "
                              "would you want to see tracebacks?")
 
+        preprocess_options = []
+        for param in param_iter:
+            text = ["<b>%s:</b> %s" % (k, v)
```

Comment:
Very minor, but this can be a generator expression. The benefit is that the code on 187-188 will do a traversal of `param.values`, and then the join on 191 will do a traversal of the resulting list. A generator expression here will reduce this to a single traversal of the data. 

```python
text = ("<b>%s:</b> %s" % (k, v) for k, v in viewitems(param.values))
```